### ISSUE TITLE
Feature Request: Add a reader view

### ISSUE THREAD

**Issue by bhdouglass** [ubports/morph-browser#170]
Many mobile browsers have a "reader view" for reading articles in a distraction free environment. Mozilla has a stand alone library for Firefox's reader view called "Readability": https://github.com/mozilla/readability
Readability could be used to help make the reader view.

**Comment by cibersheep** [comment]
For now the only workaround would be PockIt
https://open-store.io/app/pockit.turan-mahmudov-l

I love Firefox ability to take out the css :)

**Comment by mateosalta** [comment]
might be worth a look on how qutebrowser 
 does it: https://github.com/qutebrowser/qutebrowser/blob/master/misc/userscripts/readability

**Comment by mateosalta** [comment]
research:
https://doc.qt.io/qt-5/qtwebengine-webenginewidgets-contentmanipulation-example.html
https://www.slideshare.net/qtbynokia/special-effects-with-qt-graphics-view

**Comment by Rebecca-du-58** [comment]
Yes, I want a reader view too, and with large print, and a dark mode. pleaaaaaase!

**Comment by niutech** [comment]
There are also similar projects:
https://github.com/luin/readability
https://github.com/ZachSaucier/Just-Read (extension)
https://github.com/ankit/stylebot (extension)
https://gist.github.com/akaleeroy/855be4a4a690ff9dafa05234f2f35928 (bookmarklet)
https://github.com/viktor-evdokimov/bookmarklet-readability (bookmarklet)
https://github.com/ejucovy/readability/tree/gh-pages (bookmarklet)

Is anybody willing to incorporate any of these in Morph Browser?
